### PR TITLE
Class namespace usage

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

**Comment by adamweeks** [comment]
> Are there new scss processing requirements for the component? Or will all that be handled at build time before it's published?

@andrewholsted no. We will still be publishing css for consumption.

File: src/components/WebexActivityStream/TimeRuler.scss [hunk 1]
```diff
@@ -1,85 +1,5 @@
-// For Storybook, so that activity stream loads content properly
```
Comment (adamweeks) [inline src/components/WebexActivityStream/TimeRuler.scss]: None of these styles had to do with the time ruler, so I moved them to `src/components/WebexActivityStream/WebexActivityStream.scss`

File: src/components/WebexMeetingInfo/WebexMeetingInfo.scss [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
   flex-direction: column;
   text-align: center;
-}
 
-.meeting-info h2 {
-  font-size: 28px;
-  font-family: $brand-font-extra-light;
-  color: $md-gray-100;
-}
+  h2 {
```
Comment (InteractiveTimmy) [inline src/components/WebexMeetingInfo/WebexMeetingInfo.scss]: good catch for nesting opportunity.

**Pull request merged**